features/step_definitions/images/image_create_steps.rb

```diff
@@ -30,12 +30,15 @@
 end
 
 When(/^I create new glance image based on jeos$/) do
+  imagedir = File.expand_path "~/cct-images/"
```

Comment:
It doesn't create the directory on the users workstation. This is just creating a path. The directory itself is create remotely in the ```download_files``` remote command.